dataGenerator.js

```diff
@@ -41,12 +22,54 @@ let photoCounter = 1;
 
 let startTime = new Date;
 let endTime;
+let numberToWrite = 10000000;
+
+let reviews1 = '';
```

Comment:
This seems to be very wet, the code looks like it works. But I am convinced that it may be done cleaner and dryer.
With this in mind, you may want to time box 2 or so hours to make these into a loop so you can still drop data when you are finished, but not have as wet code. 
If you choose to keep the code as is you may want to consider how to explain why you are keeping it this way. Ideas: MVP / time constraint, or looping it would require nested for loops which over load the heap or increase to generate the data. 

For ideas you may want to review and mimic some of my code. generateStyleData.js
![image](https://user-images.githubusercontent.com/63127566/92147571-906cf500-edd8-11ea-8fbc-e6844aaa8c3f.png)

![image](https://user-images.githubusercontent.com/63127566/92147523-83e89c80-edd8-11ea-8e67-c6d556e09b4a.png)